src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/PropertyBasedEmbeddingProviderConfig.java

```diff
@@ -61,6 +61,9 @@ interface ModelConfig {
       @JsonProperty
       String name();
 
+      @JsonProperty
+      Integer vectorDimension();
```

Comment:
Thought @amorton was suggesting to have it as array.

Reply:
In that case, I don't know which dimension I should populate if the user does not provide the dimension. We don't know if we will support models with multiple dimensions so I leave it like that. We could change in the future if we support it.